src/base/bittorrent/session.cpp

```diff
@@ -3816,22 +3817,30 @@ bool Session::addMoveTorrentStorageJob(TorrentHandleImpl *torrent, const QString
         if (iter != m_moveStorageQueue.end()) {
             // remove existing inactive job
             m_moveStorageQueue.erase(iter);
+            LogMsg(tr("Cancelled to move \"%1\" from \"%2\" to \"%3\".").arg(torrent->name(), currentLocation, iter->path));
```

Comment:
@glassez:  Technically, this phrase is improper English grammar when used with `Cancelled`.  Please use… 

> Cancelled moving \"%1\" 

…Instead. 

The other log messages are all fine.  Proper English phrasing can be remarkably inconsistent.

Reply:
Thank you! Will fix.

Reply:
>The other log messages are all fine.

@Chocobo1, @thalieht, are you OK with updated messages?

Reply:
Yeah looks good to me.